AWSAppSyncUnitTests/AWSAppSyncCacheConfigurationTests.swift

```diff
@@ -114,16 +114,17 @@ class AWSAppSyncCacheConfigurationTests: XCTestCase {
         )
         let cacheConfig = try AWSAppSyncCacheConfiguration(useClientDatabasePrefix: true, appSyncServiceConfig: serviceConfigWithFooPrefix)
         let clientConfig = try AWSAppSyncClientConfiguration(appSyncServiceConfig: serviceConfigWithFooPrefix, cacheConfiguration: cacheConfig)
-        let client = try AWSAppSyncClient(appSyncConfig: clientConfig)
-        let clientb = try AWSAppSyncClient(appSyncConfig: clientConfig)
+        let clientA = try AWSAppSyncClient(appSyncConfig: clientConfig)
```

Comment:
can we use `let _ = ....`?

Reply:
can't this triggers dealloc code

Reply:
That should not affect the `try` logic test. The `dealloc` would be done once the init for `AWSAppSyncClient` is complete I believe.

Reply:
It doesn't affect clientA or clientB it affects the last client construction which expects a failure because the prefix is used. By using the `_`, the clients are de-allocated before the final client can test a failure case.